webrtc.html

```diff
@@ -1297,15 +1294,13 @@
               </li>
 
               <li>
-                <p>If <var>candidate</var> is missing values for both <var>sdpMid</var> and
-                <var>sdpMLineIndex</var>, return a promise rejected with a
-                <code>TypeError</code>.</p>
+                <p>Let <var>candidate</var> be the methods argument.</p>
               </li>
 
               <li>
-                <p>If the candidate could not be successfully applied, return a promise
-                rejected with a <code>DOMException</code> object whose <code>name</code>
-                attribute has the value <code>OperationError</code>.</p>
+                <p>If <var>candidate</var> is missing values for both <var>sdpMid</var> and
+                <var>sdpMLineIndex</var>, return a promise rejected with a
+                <code>TypeError</code>.</p>
```

Comment:
The above text duplicates what's said in the RTCIceCandidate constructor. We could specify it as 'construct an RTCIceCandidate' and don't have to care about what the argument was from this point on. I noted that the RTCSessionDescription constructor has a note about it being deprecated. Perhaps the RTCIceCandidate constructor has the same status.